avaframe/runScripts/runComparisonModules.py

```diff
@@ -134,6 +136,7 @@
             else:
                 cfgAimec['FLAGS']['flagMass'] = 'False'
             cfgAimec['AIMECSETUP']['comModules'] = 'com1DFAOrig|com1DFA'
+            cfgAimec['AIMECSETUP']['resType'] = 'ppr'
```

Comment:
why do you hardcode this? why not reed the ini file?

Reply:
this if for the report generation of the comparison report - where we use the locals for com1DFA and com1DFAOrig for this runScript to perform the ava simulations, but to run aimec and generate the report we use some fixed settings in the  runScripts - that is for this runScript one doesn't have to change the aimec configuration - but it is set how we defined it for this test

Reply:
mm still not convinced